### PR TITLE
Fix file exporter

### LINKED ISSUE #140: File exporter creates many incomplete files of size 1086B

### PR DESCRIPTION
**Pull request by joshbetz**
The file exporter was having some problems downloading partial files.
Adding the request library to take care of this. It would be nice if we
could use superagent, but we already tried that and it doesn't seem to
work :( Maybe we should switch to request for everything?

Fixes #140

### CODE REVIEW

File: src/bin/vip-files.js [hunk 1]
```diff
@@ -60,15 +61,10 @@ program
...
-									https.get( filedata, download => {
-										download.pipe( newFile );
-										download.on( 'end', () => {
-											bar.tick();
-											newFile.close( callback );
-										}).on( 'error', err => {
-											fs.unlink( dest );
-											callback( err );
-										});
+									var url = 'https://' + site.domain_name + file.file_path;
```
Comment (mjangda) [inline src/bin/vip-files.js]: The request is a little different here. Previously we were hitting `files.vipv2.net` with a `Host` header pointing to the site domain. This just hits the site domain directly. Is that change intentional? Can we remove the `filedata` var?

Comment (joshbetz) [inline src/bin/vip-files.js]: > Previously we were hitting files.vipv2.net with a Host header pointing to the site domain.

When I made the request against files.vipv2.net, the request library tried to verify the files.vipv2.net cert against the domain in the Host header. This could potentially introduce an issue if the domain_name is not a go-vip.co domain and the DNS has been pointed away from VIP Go.

>  Can we remove the filedata var?

Good call. Removed.

**Comment by joshbetz** [comment]
Trying a simpler fix
